app/src/main/res/values/colors.xml

```diff
@@ -13,4 +13,7 @@
   <color name="white">#FFFFFF</color>
   <!-- AUDIO COMPONENT -->
   <color name="audioComponentBackground">@color/oppiaDarkBlue</color>
+
+  <color name="blue_100">#0F0086FB</color>
```

Comment:
We are following camel case is't?

Reply:
yes, but if there are more than one lighter shade of same color how to name it.